### PR TITLE
Show starting score of each round

### PR DESCRIPTION
**Pull request by davidtcalabrese**
image

Also adds favicon.

### CODE REVIEW

**Comment by pwinckles** [comment]
What do you think about changing `Points` to `Points Gained`?

File: haggis.js [hunk 1]
```diff
@@ -369,6 +372,12 @@ function renderStatsAsHtmlString(tableId, stats) {
     output += `    <th>${player1}</th>\n`;
     output += `    <th>${player2}</th>\n`;
     output += "  </thead>\n";
+    output += "  <tr>\n";
+    output += "    <td>Starting Score </td>\n";
+    output += `    <td>${round.startingScore[player2] ? round.startingScore[player1] : 0}</td>\n`;
```
Comment (pwinckles) [inline haggis.js]: I think you can use the `??` operator here like: `round.startingScore[player1] ?? 0`

File: haggis.js [hunk 2]
```diff
@@ -554,3 +563,34 @@ function extractTableId(doc) {
...
   return a - b;
 }
+
+function addPointsPerRound(game) {
+    const ptsPerRound = getTotalPointsAfterEachRound(game.rounds);
+    
+    game.rounds.forEach((_, i) => {
+        if (i >= game.rounds.length - 1) { 
+            return;
+        }
```
Comment (pwinckles) [inline haggis.js]: You don't have to change this, but, personally, I think an og-style for loop is more clear here than the `forEach`

File: haggis.js [hunk 3]
```diff
@@ -554,3 +563,33 @@ function extractTableId(doc) {
 function sortNumeric(a, b) {
   return a - b;
 }
+
+function addPointsPerRound(game) {
+    const ptsPerRound = getTotalPointsAfterEachRound(game.rounds);
+    
+    for (let i = 0; i < game.rounds.length; i++) {
+        if (i >= game.rounds.length - 1) {
```
Comment (pwinckles) [inline haggis.js]: I don't think you need this if conditional if you change the loop to be `i < game.rounds.length - 1`

Comment (davidtcalabrese) [reply haggis.js]: good call.

**Pull request merged**